src/components/molecules/ChatList/ChatList.tsx

```diff
@@ -31,7 +36,17 @@ const ChatList: React.FC<ChatListProps> = ({
   emptyListMessage,
   deleteMessage,
 }) => {
-  const usersById = useSelector(chatUsersSelector);
+  const usersById = useSelector(venueChatUsersSelector) ?? {};
+  const venueId = useVenueId();
+  const venueChatUsersQuery = useMemo<ReduxFirestoreQuerySetting>(
+    () => ({
+      collection: "users",
+      where: ["enteredVenueIds", "array-contains", venueId],
+      storeAs: "venueChatUsers",
+    }),
+    [venueId]
```

Comment:
We shouldn't need to memo this as `useFirestoreConnect` does a deep compare anyway. Probably won't hurt though.